eng/pipelines/templates/steps/azd-login.yml

```diff
@@ -0,0 +1,20 @@
+parameters:
+  SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+  AzdDirectory: "cli/azd"
```

Comment:
I can also just add logic here now that uses the `azd` in the CWD if it's there and if not falls back to just running the version from `$PATH` (if it's there)?  Should be easy enough.  Let me know.

Reply:
Let's do that. We do probably want an explicit parameter to avoid accidents where if `azd` was installed, we end up not testing the new private bits.

So I'd suggest to default `AzdDirectory: ""` and use this as the `$PATH` case.